test/rekt/features/sinkbinding/feature.go

```diff
@@ -120,6 +120,31 @@ func SinkBindingV1Job(ctx context.Context) *feature.Feature {
 	return f
 }
 
+func SinkBindingV1DeploymentTLS(ctx context.Context) *feature.Feature {
+	sbinding := feature.MakeRandomK8sName("sinkbinding")
+	sink := feature.MakeRandomK8sName("sink")
+
+	f := feature.NewFeatureNamed("SinkBinding V1 Deployment test with TLS")
+
+	f.Setup("install sink", eventshub.Install(sink, eventshub.StartReceiverTLS))
+
+	f.Requirement("install SinkBinding", func(ctx context.Context, t feature.T) {
+		d := service.AsDestinationRef(sink)
+		d.CACerts = eventshub.GetCaCerts(ctx)
+
+		cfg = append(cfg, sinkbinding.WithSink(d))
+		sinkbinding.Install(sbinding, cfg...)(ctx, t)
+	})
+	f.Requirement("SinkBinding goes ready", sinkbinding.IsReady(sbinding))
+	f.Stable("SinkBinding as event source").
+		Must("delivers events on sink with ref",
+			eventasssert.OnStore(sink).
+				Match(eventasssert.MatchKind(eventshub.EventReceived)).
+				AtLeast(1),
+		)
+	return f
+}
```

Comment:
This is a nice start!

we need to configure the SinkBinding correctly, for example, now there is no subject (see SinkBindingV1Deployment function line 42 for an example), something like this (I didn't check it in depth but this is the idea)

```suggestion
func SinkBindingV1DeploymentTLS(ctx context.Context) *feature.Feature {
	sbinding := feature.MakeRandomK8sName("sinkbinding")
	sink := feature.MakeRandomK8sName("sink")
	subject := feature.MakeRandomK8sName("subject")

	f := feature.NewFeatureNamed("SinkBinding V1 Deployment test with TLS")

	f.Setup("install sink", eventshub.Install(sink, eventshub.StartReceiverTLS))
	
	f.Setup("install a deployment", deployment.Install(subject, heartbeatsImage,
		deployment.WithEnvs(map[string]string{
			"POD_NAME":      "heartbeats",
			"POD_NAMESPACE": env.Namespace(),
		})))
	extensions := map[string]string{
		"sinkbinding": extensionSecret,
	}
	cfg := []manifest.CfgFn{
		sinkbinding.WithExtensions(extensions),
		deployment.AsTrackerReference(subject),
	}

	f.Requirement("install SinkBinding", func(ctx context.Context, t feature.T) {
		d := service.AsDestinationRef(sink)
		d.CACerts = eventshub.GetCaCerts(ctx)

		cfg = append(cfg, sinkbinding.WithSink(d))
		sinkbinding.Install(sbinding, cfg...)(ctx, t)
	})
	f.Requirement("SinkBinding goes ready", sinkbinding.IsReady(sbinding))
	f.Stable("SinkBinding as event source").
		Must("delivers events on sink with ref",
			eventasssert.OnStore(sink).
				Match(eventasssert.MatchKind(eventshub.EventReceived)).
				AtLeast(1),
		)
	return f
}
```